server/event_handler.py

```diff
@@ -60,13 +90,13 @@ async def __call__(self, request: EventRequest, room_code: str) -> None:
                                 ),
                             )
                             await self.client.send(response)
```

Comment:
This is sending a tuple[EventResponse] but `send` expects an EventResponse. Need to fix for all of the responses below.

Reply:
It looks like it's a problem only for that particular response.